src/app/components/evaluation-info/evaluation-info.component.html

```diff
@@ -19,10 +19,15 @@ <h1 class="nowrap">Move:&nbsp;</h1>
       <mat-icon class="icon-color" svgIcon="ic_chevron_left"></mat-icon>
     </button>
     <h1 class="nowrap" [ngStyle]='getMoveStyle()'>{{ displayedMoveNumber }}</h1>
-    <button aria-label="Increment Displayed Move" mat-icon-button [disabled]="isIncrementDisabled()" (click)="incrementDisplayedMove()" title="Increment Move">
+    <button aria-label="Increment Displayed Move" mat-icon-button [disabled]="isIncrementDisabled() || displayedMoveNumber === currentMoveNumber" (click)="incrementDisplayedMove()" title="Increment Move">
       <mat-icon class="icon-color" svgIcon="ic_chevron_right"></mat-icon>
     </button>
-    <h1 class="space-right">of&nbsp;&nbsp;&nbsp;{{ getMaxMoveNumber() }}</h1>
+    <h1>&nbsp;of&nbsp;&nbsp;&nbsp;{{ getMaxMoveNumber() }}</h1>
+    <div class="move-div" *ngIf="selectedEvaluationId && displayedMoveNumber === currentMoveNumber">
+      <button aria-label="Next Move" mat-button class=" blue-button mat-focus-indicator move-button background-button mat-raised-button mat-button-base mat-user ng-star-inserted" [disabled]="isIncrementDisabled()" (click)="incrementDisplayedMove()" title="Next Move">

```

Comment:
instead of disabling, I would use *ngIf to only display when user canIncrement and displayed move equals current move

Reply:
Already made recommended changes, let me know if anything else is needed.